app/browser/reducers/ledgerReducer.js

```diff
@@ -188,6 +188,7 @@ const ledgerReducer = (state, action, immutableAction) => {
       }
     case appConstants.APP_NAVIGATOR_HANDLER_REGISTERED:
       {
+        // TODO will this be changed when switching to BAT
         const hasBitcoinHandler = (action.get('protocol') === 'bitcoin')
```

Comment:
What action do we need to take here? does BAT have a handler? maybe we should remove this and create a new issue to address

Reply:
This is TODO for me, because I am not sure that we need bitcoin handler anymore. We don't use it anymore. @mrose17 can we remove this?

Reply:
just received confirmation from @mrose17 that we still need to keep it

Reply:
removing TODO tag